### PR TITLE
Added Continuous Integration via GitHub Actions

### PR DESCRIPTION
**Pull request by DhruvSondhi**
Hello, 
This PR aims to implement #22 
GitHub Actions has been set up for the same & the configuration has been tested within the fork.
Thanks :)

### CODE REVIEW

**Comment by DhruvSondhi** [comment]
@astrojuanlu please check the PR for any changes :)

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+name: Poliastro Website CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the sources branch
+  push:
+    branches: 
+      - sources
+      - 'content'
```
Comment (astrojuanlu) [inline .github/workflows/main.yml]: I wonder why the second branch name needs quotes?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    needs: build
+    strategy:
+      matrix:
+        python-version: [3.9]
+
+    steps:
+      - name: Displaying Python Versions
+        run: python -c "import sys; print(sys.version)"
+
+    # # Uncomment for deployment
```
Comment (astrojuanlu) [inline .github/workflows/main.yml:63]: My suggestion is that you try this on your fork, to make sure it works. If you did already, could you please send a link to the relevant pipelines and their results?

**Review by astrojuanlu — COMMENTED** [review]
Hi @DhruvSondhi , thanks for putting this forward and sorry for the delay! I left a couple of superficial comments because I understand that this still needs some work, but please let me know otherwise.

File: .github/workflows/main.yml (hunk 1 above)
Comment (DhruvSondhi) [reply .github/workflows/main.yml]: I have tried to implement it, in such a way that if any changes to the `content` folder of the repository in any branch (PR based or else wise) are made, the pipeline will trigger the build execution (for the same please check this [link](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-branches-and-tags))

File: .github/workflows/main.yml (hunk 2 above)
Comment (DhruvSondhi) [reply .github/workflows/main.yml:63]: Sure, I have tried it on my fork & have successfully triggered the pipeline execution via GitHub Actions ... the link for the same is [here](https://github.com/DhruvSondhi/poliastro.github.io/actions) ... Please check the same :) 

*(PS: I have left the `deploy` job for configuration ... please make the necessary changes so as to meet the requirements 😄)*

Comment (DhruvSondhi) [reply .github/workflows/main.yml:63]: The most recent run for the pipeline can be found [here](https://github.com/DhruvSondhi/poliastro.github.io/actions/runs/594940098)

Comment (DhruvSondhi) [reply .github/workflows/main.yml:63]: You can check the other changes [here](https://github.com/DhruvSondhi/poliastro.github.io)

File: .github/workflows/main.yml (hunk 1 above)
Comment (astrojuanlu) [reply .github/workflows/main.yml]: The link you sent refers to branches. I think this is what you mean:

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-paths

In any case, let's remove the `'content'` line (and the other below this one) so we only trigger builds on the `sources` branch and pull requests.

File: .github/workflows/main.yml (hunk 2 above)
Comment (astrojuanlu) [reply .github/workflows/main.yml:63]: I see on https://github.com/DhruvSondhi/poliastro.github.io/actions/runs/594940098 that this was built on several Python versions. Let's not do that - the poliastro website is not a library that will be reused, so we don't need to make sure that it runs in several Python versions.

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
Sorry for the super long delay 🙏🏽 I left a couple of comments, but this is on the right path!

**Comment by DhruvSondhi** [comment]
Thank you very much for the reply :) ... It is awesome having you back 😄

**Review by astrojuanlu — APPROVED** [review]
Thanks again @DhruvSondhi ! :tada: I know this is not yet complete but we can make incremental improvements

**Comment by DhruvSondhi** [comment]
I will keep this updated ... Very happy to have you back @astrojuanlu 😄

**Pull request merged**